### PR TITLE
Update ja.json

### PR DESCRIPTION
**Pull request by midoritsuki**
Translate badge

### CODE REVIEW

File: lang/ja.json [hunk 1]
```diff
@@ -397,84 +397,84 @@
...
           "compass_diamond": {
-            "name": "Diamond Expedition Completionist",
-            "description": "Complete expeditions in 95% of available locations"
+            "name": "ダイアモンドドリームランナー",
+            "description": "ドリームラリーで選ばれる世界の内95%の世界のラリーを完了させる"
           },
           "crushed": {
-            "name": "Crush",
-            "description": "Watch Urotsuki get smooshed"
+            "name": "クラッシュ,
```
Comment (Flashfyre) [inline lang/ja.json:460]:
```suggestion
            "name": "クラッシュ",
```

**Review by Flashfyre — CHANGES_REQUESTED** [review]

File: lang/ja.json [hunk 2]
```diff
@@ -397,84 +397,84 @@
       }
     },
     "badges": {
-      "null": "No Badge",
+      "null": "バッジなし",
       "locked": "???",
```
Comment (Flashfyre) [inline lang/ja.json:401]:
```suggestion
      "locked": "？？？",
```

File: lang/ja.json [hunk 3]
```diff
@@ -397,84 +397,84 @@
...
       "location": "Location:&nbsp;",
```
Comment (Flashfyre) [inline lang/ja.json:402]:
```suggestion
      "location": "場所：",
```
Should it be like this?

**Review by Flashfyre — CHANGES_REQUESTED** [review]

File: lang/ja.json (hunk 3 above)
Comment (midoritsuki) [reply lang/ja.json:402]: yes
thanks

**Pull request merged**